.github/mergify.yml

```diff
@@ -3,9 +3,8 @@ pull_request_rules:
     conditions:
       - author~=^dependabot\[bot\]$
       - check-success~=lint
-      - check-success~=build
       - check-success~=test
-      - title~=Bump [^\s]+ from ([\d]+)\..+ to \1\.
+      - dependabot-update-type = version-update:semver-minor
```

Comment:
what about patches?

Reply:
updated to auto-update for all non-major